README.md

```diff
@@ -26,6 +26,12 @@
 
 Install the plugin with your preferred package manager:
 
+### [lazy.nvim](https://github.com/folke/lazy.nvim)
+
+```lua
+{ "folke/neoconf.nvim" }
```

Comment:
I believe `opts` shouldn't be provided by default, as it's important that `neoconf.nvim` explicitely set up **before** `nvim-lspconfig`